client/src/react-app-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="react-scripts" />
```

Comment:
Not sure if this is required, was created automatically. Pls advise

Reply:
Apparantly it says not to remove this. I've never seen this file before but okay.